jobs/delorean/jenkinsfiles/discovery/rc/github/Jenkinsfile

```diff
@@ -18,8 +18,58 @@ def templatesForkGitUrl = "https://github.com/${templatesForkOrg}/${projectRepo}
 def templatesDir = params.templatesDir
 def latestPreRelease
 
+String brewContainerRegistry = "brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888"
+String rhContainerRegistry = "registry.redhat.io"
+String rhContainerRegistry2 = "registry.access.redhat.com"
+String clusterContainerRegistryPrivate = "docker-registry.default.svc:5000"
```

Comment:
do  you think we need to use those vars as parameters? or is it over engineering things?

Reply:
Can't see the need to add them as parameters tbh, it's a pretty well known list of registries, and won't really change. Prefer to keep the parameters to a minimum.